addons/account/static/src/components/actionable_errors/actionable_errors.js

```diff
@@ -11,6 +11,15 @@ export class ActionableErrors extends Component {
     async handleOnClick(errorData){
         this.env.model.action.doAction(errorData.action);
     }
+
+    get sortedActionableErrors() {
+        let data = this.props.record.data[this.props.name];
+        return Object.fromEntries(Object.entries(data).sort((a, b) =>
+              a[1]["level"] == "critical" && !b[1]["level"] == "critical" ? -1
+            : !a[1]["level"] == "critical" && b[1]["level"] == "critical" ? 1
+            : a[0].localeCompare(b[0])
+        ));
```

Comment:
```suggestion
    get sortedActionableErrors() {
        let data = this.props.record.data[this.props.name];
        return Object.fromEntries(Object.entries(data).sort((a, b) => (
            DISPLAY_ORDER.indexOf(a[1]["level"] || "warning") - DISPLAY_ORDER.indexOf(b[1]["level"] || "warning"))
        ));
    }

```
personally, I prefer something like this, where `DISPLAY_ORDER` is a `const`
```js
const DISPLAY_ORDER = [
    'danger',
    'warning',
    'info',
];
```
or alternatively it could be a method on the component.

Reply:
I'd prefer a method, readability counts, a lot.

```js
        return Object.fromEntries(Object.entries(this.comparisonFunction));
    }
    
    function comparisonFunction(a, b) {
        let order = ['danger', 'warning', 'info'];
        let a_level = a[1].level || "warning";
        let b_level = b[1].level || "warning";
        return order.indexOf(a_level) - order.indexOf(b_level);
    }
```

I'm not good at JS, I wrote this without a case, it's just an idea.